code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -1300,6 +1303,8 @@
 		animate(item)
 		if(item.material)
 			item.setMaterialAppearance(item.material)
+		item.UpdateIcon()
+		item.dropped(holder)
```

Comment:
this seems a bit strange to call directly

Reply:
where would be better to call it?